Rocks.Wasabee.Mobile.Core.Ui/Views/AgentVerification/SubViews/AgentVerificationStep3View.xaml

```diff
@@ -0,0 +1,102 @@
+                      <StackLayout Spacing="15">
+                          <Label Text="Post your token in the Activity feed of Ingress Community forums."/>
+
+                          <Button Text="Open Community Forums"
+                                  HorizontalOptions="Center"
+                                  CornerRadius="5"
+                                  TextColor="White"
+                                  Command="{Binding OpenCommunityCommand}">
+                              <Button.Background>
+                                  <LinearGradientBrush StartPoint="0,0" EndPoint="0,1">
+                                      <GradientStop Color="#79409E"
+                                                    Offset="0.0" />
+                                      <GradientStop Color="#5A2D92"
+                                                    Offset="1.0" />
+                                  </LinearGradientBrush>
+                              </Button.Background>
+                          </Button>
+                      </StackLayout>
+                      
+                      <StackLayout Spacing="15" IsVisible="{Binding HasOpenedCommunity}">
+                          <Label>
+                              <Label.FormattedText>
+                                  <FormattedString>
+                                      <Span Text="Once you've posted the token to the community Forums, click the button below to start the verification process."/>
+                                  </FormattedString>
+                              </Label.FormattedText>
+                          </Label>
+
+                          <Button Text="Verify me"
+                                  HorizontalOptions="Center"
+                                  Style="{DynamicResource WasabeeStyleMainActionButton}"
+                                  Command="{Binding RefreshStatusCommand}"/>
+
+                          <ActivityIndicator IsRunning="True"
+                                             IsVisible="{Binding IsBusy}"/>
+                      </StackLayout>
+                  </StackLayout>
+
+                  <StackLayout Grid.Row="1" Padding="10" VerticalOptions="Center" IsVisible="{Binding IsVerified}">
+                      <Label HorizontalTextAlignment="Center">
+                          <Label.FormattedText>
+                              <FormattedString>
+                                  <Span Text="Well done !"/>
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                                  <Span Text="Your Wasabee account in now verified against Community Forums !"/>
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                                  <Span Text="{x:Static system:Environment.NewLine}"/>
+                                  <Span Text="Your profile now displays a 🐝 !"/>
```

Comment:
here

Reply:
77: "Your Wasabee account is now verified with the Community forums."

75, 80: No space before "!" in English